src/ui/panels/inventory.js

```diff
@@ -129,11 +124,11 @@ export class Inventory {
                             );
                     }
                 }
-            } else {
-                return characterData
-                    .getInventory()
-                    .indexOf(itemClasses[i].constructor.name);
             }
+        } else {
+            return characterData
+                .getInventory()
+                .findIndex((item) => item.item.includes(keyword));
         }
 
         return -1;
```

Comment:
this will never be reachable now. can remove

Reply:
Alrighty

Reply:
Actually looking at it I think that -1 is needed. There are several conditions in the upper part of the function that if they are false it would go to that -1 at the end.

Reply:
Ah yes I see what you mean